contrib/cirrus/runner.sh

```diff
@@ -345,6 +390,32 @@ dotest() {
         |& logformatter
 }
 
+# Helper: returns true if the only PR diffs are in the given (arg) directories
+function _only_changes_are_in() {
+    local head base diffs
+
+    # Cirrus sets this for PRs but not cron. In cron, we never want to skip.
+    if [[ -z "$CIRRUS_CHANGE_IN_REPO" ]]; then
```

Comment:
Ya this is good.  Probably should stick in a `|| [[ -n "$CIRRUS_TAG" ]]` here too.